cfgov/prepaid_agreements/jinja2/prepaid_agreements/search_results.html

```diff
@@ -0,0 +1,174 @@
+                    {{ total_count }} total products
+                </h3>
+            </div>
+        {% else %}
+            <div class="results_count no-results" data-results-count="0">
+                <h3>No results match your search.</h3>
+                <p>Try adjusting your filters or searching for different terms.</p>
+            </div>
+        {% endif %}
+    </div>
+    
+    {% if filters | length %}
+    
+        <div class="filters_applied">
+            <div class="filters_tags">
+                <span class="filters_applied-label">
+                Filters applied:
+            </span>
+                {% set filters_to_remove = [] %}
+                {% for filter in filters %}
+                    {% if filter in valid_filters %}
+                        {% for val in filters[filter] %}
+                            {% set href = remove_url_parameter(request, {filter: [val]}) %}
+                            {{ tag.render({
+                                'label': val,
+                                'value': val,
+                                'behavior': 'clear-filter',
+                                'href': href if href else '.'
+                            }) }}
+                        {% endfor %}
+                    {% endif %}
+                {% endfor %}
+
+                {% set clear_url = remove_url_parameter(request, filters) %}
+                <a class="a-btn a-btn__link a-btn__warning a-micro-copy results_filters-clear u-mb10"
+                        data-js-hook="behavior_clear-all" href="{{ clear_url if clear_url else '.' }}">
+                    Clear all filters
+                </a>
+            </div>
+        </div>
+    {% endif %}
+
+    <div class="results_list">
+        {% for result in results %}
+            {{ search_item.render( result ) }}
+        {% endfor %}
+    </div>
+
+    <div class="results_paginator">
+        {% if paginator.num_pages > 1 %}
+            <div class="block block__flush-top">
+                {{ pagination.render( paginator.num_pages, current_page, '', '', 'Previous', 'Next' ) }}
+            </div>
+        {% endif %}
+    </div>
+
+    <div class="database-disclaimer">
+        <div class="o-well">
+            <h4>Database disclaimer</h4>
+            <p>We will display the prepaid agreements in this database as the respective issuers submitted them. The CFPB is not responsible for the content of the agreements, including any discrepancies between an agreement as presented in this database and the agreement as offered to the public, or for any omissions or other errors in the agreement as submitted by the issuer.</p>
+            <p>The agreements on file will have general terms and conditions, pricing, and fee information. They are not specific to an individual's account information.</p>
+            <p>If you have questions about the agreements themselves, contact the prepaid issuer directly.</p>
+        </div>
+    </div>
+</div>
```

Comment:
TRIVIAL: Add ending newline.

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/5207/commits/7371cd90fac1928b166a496101f3bb76ed858b96